src/types/signIn.ts

```diff
@@ -0,0 +1,7 @@
+import { Resource } from 'types/resource';
+
+export interface SignIn extends Resource {
```

Comment:
Should this resource be named `Token` instead? As it contains the token information 🤔 
```suggestion
export interface Token extends Resource {
```

Reply:
@hoangmirs fixed in [90287e8](https://github.com/manh-t/react-survey/pull/49/commits/90287e88fb7a8b870a60719bb83fc9a8264dfcea)